mean/modules/survey/views/p20_30.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://code.angularjs.org/1.4.3/angular.js"></script>
+    <link rel="stylesheet" href="survey.css" />
+    <script src="gen%20info%20JS.js"></script>
+  </head>
+
+<!-- Insert header here -->
+    <body>
+        <h3>Diagnosis Confirmation</h3>
+        <hr>
+        <p>Please confirm the diagnoses for the member</p>
+        <table class="info">
+            <tr>
+                <td class="col-md-1">
+                  <p>Diagnosis</p>
+                </td>
+                <td class="col-md-2">
+                  <p>Active</p>
+                </td>
+                <td class="col-md-2">
+                  <p>Assessment</p>
+                </td>
+                <td class="col-md-2">
+                  <p> Plan </p>
+                </td>
+                <td class="col-md-3">
+                  <p> Notes </p>
+                </td>
+            </tr>
+            <tr>
+                <td class="col-md-1">
+                  <p>Angina - Stable</p>
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="angina" />
+                </td>
+                <td class="col-md-2">
+                  <input type="checkbox" name="angina_stable" />Stable <br/>
```

Comment:
If there's a break line between every one of these, first, it's an incorrect tag format, second, create a class for this and put the styling in the css